src/components/SearchGridManualLoad.vue

```diff
@@ -97,6 +99,9 @@ export default {
     _query() {
       return this.$props.query;
     },
+    isFinished() {
+      return this.currentPage * 20 >= this.$store.state.imagesCount;
```

Comment:
I'm not sure I understood the logic here. Shouldn't it compare `state.images.length` with `state.imagesCount`?

Reply:
So for some reason, sometimes there are fewer images in the array than the image Count even when there are no more images to load.
So the "No more Images" never shows and the link still works.
But checking the number of pages multiplied by the default number of images works.

Reply:
Do you know of a few search terms where this reproduces? This seems like a bug on the API, right @aldenstpage ?

Reply:
You can search "meangirls", 
The imageCount is 94 but it returns 85 images

Reply:
That looks like a bug in the API.  I'll file an issue there and when it's fixed, we can fix it here

Reply:
Although for now, I think we can fix it using the `page_count` value that comes in the response to the search request as you can see [here](https://api.creativecommons.engineering/image/search?q=meangirls). Saving that on the store and then checking if the `currentPage === pageCount` should be good enough